### PR TITLE
Fix comments and some of documentation

### PR DESCRIPTION
**Pull request by astrikos**

### CODE REVIEW

File: ripe/atlas/tools/commands/probes.py [hunk 1]
```diff
@@ -14,9 +14,10 @@ class Command(BaseCommand):
 
     NAME = "probes"
 
-    DESCRIPTION = "Find probes fullfiling specific criteria based on given filters."
+    DESCRIPTION = "Fetches and prints probes fullfiling specified criteria based on given filters."
```
Comment (danielquinn) [inline ripe/atlas/tools/commands/probes.py]: pep8

File: ripe/atlas/tools/commands/probes.py [hunk 2]
```diff
@@ -56,40 +57,40 @@ def add_arguments(self):
...
             "--limit",
             type=int,
-            help="Limit"
+            help="Return limited amount of probes"
         )
         self.parser.add_argument(
             "--additional-fields",
             type=str,
-            help="Additional fields"
+            help="Print additional probe fields. Specify field names as given from the API as a comma separated string."
```
Comment (danielquinn) [inline ripe/atlas/tools/commands/probes.py]: pep8 violation :-(

File: ripe/atlas/tools/commands/probes.py [hunk 3]
```diff
@@ -56,40 +57,40 @@ def add_arguments(self):
...
         )
         self.parser.add_argument(
             "--aggregate-by",
             type=str,
             choices=['asn_v4', 'asn_v6', 'country_code', 'prefix_v4', 'prefix_v6'],
             action="append",
-            help="Aggregate list of probes based on all specified aggregations."
+            help="Aggregate list of probes based on all specified aggregations. Multiple aggragations supported."
```
Comment (danielquinn) [inline ripe/atlas/tools/commands/probes.py]: pep8

File: ripe/atlas/tools/commands/probes.py [hunk 4]
```diff
@@ -108,7 +109,7 @@ def add_arguments(self):
         )
 
     def run(self):
-
+        """Main function that will run on the command"""
```
Comment (danielquinn) [inline ripe/atlas/tools/commands/probes.py]: I'm not convinced that this is a necessary docstring as `run()` is the main function everywhere.

File: ripe/atlas/tools/commands/probes.py [hunk 5]
```diff
@@ -149,6 +150,7 @@ def produce_ids_only(self, probes):
         return ",".join(probe_ids)
 
     def _clean_render_args(self):
+        """Clean arguments that will be used by renderer and return the kwargs structure for it"""
```
Comment (danielquinn) [inline ripe/atlas/tools/commands/probes.py]: pep8

File: ripe/atlas/tools/commands/probes.py [hunk 6]
```diff
@@ -306,6 +308,7 @@ def _clean_country_code(self):
         return country_code_args
 
     def get_aggregators(self):
+        """Builds and returns the key aggregators that will be used in the aggregation."""
```
Comment (danielquinn) [inline ripe/atlas/tools/commands/probes.py]: pep8

File: ripe/atlas/tools/commands/report.py [hunk 7]
```diff
@@ -119,10 +119,7 @@ def get_aggregators(self):
         return aggregation_keys
 
     def create_enhanced_sagans(self, results):
-        """
-        Create Sagan Result objects and add additonal Porbe attribute to each
-        one of them
-         """
+        """Create Sagan Result objects and add additonal Probe attribute to each one of them."""
```
Comment (danielquinn) [inline ripe/atlas/tools/commands/report.py]: pep8, and it should probably be `**an** additional probe`.

File: ripe/atlas/tools/renderers/probes.py [hunk 8]
```diff
@@ -62,7 +62,7 @@ def _construct_probe_line(self):
             self.probe_template += "{:<}|"
 
     def render_aggregation(self, aggregation_data, indent=""):
-        """Traverses through aggregation data and print them indented"""
+        """Recursively traverses through aggregation data and print them indented."""
```
Comment (danielquinn) [inline ripe/atlas/tools/renderers/probes.py]: pep8

**Comment by astrikos** [comment]
80 char limit forced :(

**Pull request merged**